src/cockpit/389-console/src/lib/database/databaseConfig.jsx

```diff
@@ -990,3 +990,580 @@ GlobalDatabaseConfig.defaultProps = {
+                                <div className="ds-left-indent-md">
+                                    <Grid
+                                        title={_("Set the maximum size in bytes for the Normalized DN Cache (nsslapd-ndn-cache-max-size).")}
+                                        className="ds-margin-top-xlg"
+                                    >
+                                        <GridItem className="ds-label" span={4}>
+                                            {_("Normalized DN Cache Max Size")}
+                                        </GridItem>
+                                        <GridItem span={8}>
+                                            <NumberInput
+                                                value={this.state.ndncachemaxsize}
+                                                min={1000000}
+                                                max={this.maxValue}
+                                                onMinus={() => { this.onMinusConfig("ndncachemaxsize") }}
+                                                onChange={(e) => { this.onConfigChange(e, "ndncachemaxsize", 1000000, 0) }}
+                                                onPlus={() => { this.onPlusConfig("ndncachemaxsize") }}
+                                                inputName="input"
+                                                inputAriaLabel="number input"
+                                                minusBtnAriaLabel="minus"
+                                                plusBtnAriaLabel="plus"
+                                                widthChars={10}
+                                            />
+                                        </GridItem>
+                                    </Grid>
+                                </div>
+                            </Tab>
+
+                            <Tab eventKey={5} title={<TabTitleText>{_("Advanced Settings")}</TabTitleText>}>
+                                <div className="ds-left-indent-md">
+                                    <Grid
+                                        title={_("Location for database memory mapped files.  You must specify a subdirectory of a tempfs type filesystem (nsslapd-db-home-directory).")}
+                                        className="ds-margin-top"
+                                    >
+                                        <GridItem className="ds-label" span={4}>
+                                            {_("Database Home Directory")}
+                                        </GridItem>
+                                        <GridItem span={8}>
+                                            <TextInput
+                                                value={this.state.dbhomedir}
+                                                type="text"
+                                                id="dbhomedir"
+                                                aria-describedby="dbhomedir"
```

Comment:
For some reason, when I change dbhome directory, the button still stays grey. And if I change something else and then save, dbhome dir still resets to the default - `/var/lib/dirsrv/slapd-inst/db`

And it happens only for LMDB. BDB is fine

Reply:
Ok, the button change is a copy and paste error. 

Regarding the dbhome setting, when I try changing the db home directory with bdb, an unwilling to perform message is returned, in the error logs it mentions this config cannot be changed when the server is running. IIUC then do we need a config option for changing the db home in the UI at all? 

WDYT